src/logger/transformStream.ts

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
this loads on a worker thread, and pino streams messages through it before pipelining them to the final destination (file, or pretty terminal output)

The exported types aren't good, and the pipeline doesn't know what precedes it (node.PipelineSource uses `any`) I tried to clean this up but couldn't get them all.